_data/backup.yml

```diff
@@ -41,12 +41,14 @@ websites:
     twitter: cloudapp
     img: cloudapp.png
     tfa: No
+    facebook: getcloudapp
 
   - name: CrashPlan
     url: http://www.crashplan.com
```

Comment:
They are using HTTPS now. @stephengroat, would you like update the URL here or should it be done in a separate pull request?